src/components/TokenAdministrative.js

```diff
@@ -82,17 +82,20 @@ class TokenAdministrative extends React.Component {
   /**
    * Upadte token info getting data from the full node (can mint, can melt, total supply)
    */
-  updateTokenInfo = () => {
+  updateTokenInfo = async () => {
     this.setState({ errorMessage: '' });
-    hathorLib.walletApi.getGeneralTokenInfo(this.props.token.uid, (response) => {
-      if (response.success) {
-        this.setState({
-          totalSupply: response.total,
-        });
-      } else {
-        this.setState({ errorMessage: response.message });
-      }
-    });
+
+    try {
+      const tokenDetails = await this.props.wallet.getTokenDetails(this.props.token.uid);
+      this.setState({
+        totalSupply: tokenDetails.total,
+        canMint: tokenDetails.mint.length > 0,
+        canMelt: tokenDetails.melt.length > 0,
+        transactionsCount: tokenDetails.transactions_count,
+      });
+    } catch (e) {
```

Comment:
What error is expected here?

Reply:
The request to the API might fail, the current implementation on the facades is to throw on those cases